### DISCUSSION TITLE
Request backport of Webp with lossy conversion weighs a lot more than lossless #2171 to 2.1.3

### DISCUSSION THREAD

**Discussion by rpteasolutions** [SixLabors/ImageSharp#2694] [Q&A]
Hi!

I want to ask if the changes made in #2173 (PR for issue #2171) can be backported to version 2, specifically 2.1.3?

We have a big client running multiple sites on Umbraco 10. And upgrade to Umbraco 13 is in future plans, but not near enough. At the moment a lot of unnecessary bandwidth is used and we are looking into a way to have this solved in a shorter timeline than the Umbraco 13 upgrade.

We would therefore appreciate if the changes could be backported. If it is even possible for this version.

Happy to help with testing etc.

**Comment by tiesont** [comment]
Generally, only bugfixes related to security exploits are backported. I suppose if you were to [purchase a commercial license](https://sixlabors.com/pricing/) you could reasonably ask for something like this, but even then... really should be using the currently-supported version.

_(Please note that I am not part of this project, just someone who has used this project in the past and tries to chip in on the few occasions my input might be useful)._

**Reply by antonfirsov** [reply]
Personal opinion since the monetization aspect is comletely in @JimBobSquarePants's hands:

> And upgrade to Umbraco 13 is in future plans, but not near enough

This is a completely valid situation for a large project that cannot move fast, but such a non-security backport request goes beyond the scope of the normal commercial license and should be subject of an individual business agreement if that makes sense (and is worth the trouble) for both parties.

**Reply by JimBobSquarePants** [reply]
I'm open to offers 😉
